ldcoolp/curation/inspection/readme/__init__.py

```diff
@@ -217,11 +218,12 @@ def retrieve_article_metadata(self):
         # Retrieve preferred citation. Default: ReDATA in DataCite format
         # This forces a period after the year and ensures multiple rows
         # with the last two row merged for simplicity
+        # v0.17.6 handles periods in author list (e.g., middle name initial)
         single_str_citation = self.article_dict['item']['citation']
-        str_list = [str_row + '.' for str_row in
-                    single_str_citation.replace('):', ').').split('. ')]
-        citation_list = [content for content in str_list[0:-2]]
-        citation_list.append(f"{str_list[-2]} {str_list[-1]}")
+        END_SENT = re.compile('((?<=[.!?])|(?<=\.\":)) +(?=[A-Z,0-9])')
+        citation_list = list(
+            filter(None, END_SENT.split(single_str_citation.replace('):', ').')))
+        )
```

Comment:
The regexp seems solid. I am not sure If only having straight quotation marks `"` may be enough of it you would like to include curvy `“`  quotation marks as well as in `(?<=\.\["“]:))` .

One suggestion is making things more explicit and documenting the string processing to improve readability and maintainability:

```suggestion
        # Replace colon with '.' if preceded by ')'
        pre_processed_str = single_str_citation.replace('):', ').')
        # Match spaces following group 1, only if followed by group 2
        END_SENT = re.compile('((?<=[.!?])|(?<=\.\":)) +(?=[A-Z,0-9])')
        citation_list = []
        for section in END_SENT.split(pre_processed_str):
            if section:
                citation_list.append(match)
```

Reply:
I'm OK with additional documentation and breaking it apart, but some of the docs above this block is already present. For example, it says: "This forces a period after the year and ensures multiple rows with the last two row[s] merged for simplicity". So some succinctness is necessary.

Reply:
There is actually a typo. `match` is not defined.